test/apisix/Dockerfile.test.apisix.deb.focal1

```diff
@@ -8,7 +8,7 @@ ARG APISIX_VERSION
 
 ENV RUNNING_ETCD_VERSION=${ETCD_VERSION}
 
-COPY ./output/apisix_${APISIX_VERSION}-0_amd64.deb /apisix_${APISIX_VERSION}-0_amd64.deb
+COPY ./output/apisix_${APISIX_VERSION}-0~focal1_amd64.deb /apisix_${APISIX_VERSION}-0~focal1_amd64.deb
```

Comment:
The package looks strange to me. Is there any info about the name convention?

Reply:
> The package looks strange to me. Is there any info about the name convention?

@spacewander actually deb packages do not have a conventional name format for distribution section. So just in order to let users know which distribution is the package for, I refer to the openresty packages, see https://openresty.org/package/ubuntu/pool/main/o/openresty/, to define the apisix packages.

Looking forward to your insights. 😄

Reply:
Look like deb doesn't have a compulsory revision format: https://www.debian.org/doc/debian-policy/ch-controlfields.html#version

I think our revision should contain ubuntu or debian so that we can distinguish it.

Reply:
@spacewander Agreed. We could use `package_1.2.3-0~debian11_amd64.deb` for the package on Debian 11. As currently Debian is not supported yet, so we can add this in other PRs.

Regarding the packages on Ubuntu, how about keeping using the name `package_1.2.3-0~focal1_amd64`? IMO our users usually know what `focal` means. 😄

What do you think?

Reply:
Since deb can be used in debian it is better to add distribution name in the version.

Reply:
> Since deb can be used in debian it is better to add distribution name in the version.

@spacewander OK, it's reasonable. How about `package_1.2.3-0~ubuntu20.04_amd64.deb`, which is the format that some packages are using. I could hardly see any packages with the name `package_1.2.3-0~ubuntufocal_amd64.deb`.

Looking forward to your feedback.

Reply:
@spacewander

Hi, I updated the PR and set deb name to `package_1.2.3-0~ubuntu20.04_amd64.deb` for Ubuntu 20.04, so `package_1.2.3-0~ubuntu21.04_amd64.deb` will be for Ubuntu 21.04. All CIs are passed within https://github.com/imjoey/apisix-build-tools/pull/2, please take a look at your free time.